modules/nback.llvm/llvm.bal

```diff
@@ -114,19 +175,39 @@ public class Function {
 
 }
 
+# Corresponds to llvm::Module class
+public class Module {
+    private Function[] functions = [];
+    public function init() {
+    }
+
+    public function insertFunction(string name, FunctionType fnType) returns Function {
+        Function fn = new Function(name, fnType);
+        self.functions.push(fn);
+        return fn;
+    }
+
+    public function output(Output out) {
+        foreach var f in self.functions {
+            f.output(out);
+        }
+    }
+
+}
+
 # Corresponds to LLVMBuilderRef  
 public class Builder {
     private BasicBlock? currentBlock = ();
-    
+
     public function positionAtEnd(BasicBlock block) {
         self.currentBlock = block;
     }
 
-    public function alloca(IntType ty, int align) returns Value {
+    public function alloca(IntType ty, int align = 8) returns Value {
```

Comment:
Please remove default

Reply:
We only need our heap objects to be aligned so shall we use nu-aligned version of `alloca`

Reply:
What does `nu-aligned` mean?

We should only use the default if it's completely obvious what the default is? Why is alignment of 8 right for an i1?

Reply:
Fixed.

Reply:
I mean to get rid of align altogether like in https://llvm.org/doxygen/group__LLVMCCoreInstructionBuilder.html#ga3a153f9ef93ac41cf98605a28af5392f

form llvm doc
> If not specified, or if zero, the target can choose to align the allocation on any convenient boundary compatible with the type.